src/Kafka/Metrics/Metrics.stories.tsx

```diff
@@ -309,7 +324,13 @@ _Data unavailable_ empty state is shown in place of the charts with missing
 
 export const LimitsReached = Template.bind({});
 LimitsReached.args = {
-  getMetricsKpi,
+  getMetricsKpi: () =>
+    fakeApi({
+      topics: 1,
+      topicPartitions: 950,
```

Comment:
I think this example should show a number of partitions higher than the limit.

We also need a new story to show what's shown by this story currently, so a `NearingLimits` one.